dance/modules/multi_modality/match_modality/scmm.py

```diff
@@ -266,7 +269,8 @@ def getDataLoaders(dataset, batch_size, shuffle=True, device="cuda"):
 
     def forward(self, x):
         read_count = self.enc.read_count(x)
-        self._qz_x_params = self.enc(x)
+        t1, t2 = self.enc(x)
+        self._qz_x_params = tuple([t1, t2])
```

Comment:
This is a bit confusing. Isn't the return from `self.enc(x)` already a tuple of `(t1, t2)`, which is precisely the same as `tuple([t1, t2])`?